x-pack/platform/plugins/shared/onechat/server/routes/chat.ts

```diff
@@ -183,10 +183,14 @@ export function registerChatRoutes({
 
         return response.ok({
           headers: {
-            'Content-Type': 'text/event-stream',
+            // cloud compress text/* types, loosing chunking capabilities which we need for SSE
+            'Content-Type': cloud?.isCloudEnabled
+              ? 'application/octet-stream'
+              : 'text/event-stream',
```

Comment:
Let's hope this does the trick, I don't have other idea after that one

Reply:
is there a reason why we couldn't do octet-stream always?